harness/determined/cli/_util.py

```diff
@@ -133,3 +133,15 @@ def wait_ntsc_ready(session: api.Session, ntsc_type: api.NTSC_Kind, eid: str) ->
     loading_animator.clear(msg)
     if err_msg:
         raise errors.CliError(err_msg)
+
+
+# not_found_errs mirrors NotFoundErrs from the golang api/errors.go. In the cases where
+# Python errors override the golang errors, this ensures the error messages stay consistent.
+def not_found_errs(
+    category: str, name: str, session: api.Session
+) -> api.errors.BadRequestException:
+    resp = bindings.get_GetMaster(session)
+    msg = f"{category} '{name}' not found"
```

Comment:
Note -- I think the json cli formatting is doing something wonky here because I see a double quote (instead of a single quote) on my CLI:
``` 
% det workspace describe <hidden-workspace>
CLI version 0.22.0-rc0 is less than master version 0.23.2-dev0. Consider upgrading the CLI.
Failed to describe workspace: Did not find a workspace with name "testHidden2".
```

Reply:
I'm a bit confused aren't you supposed to see `not found` in the printed message if it was going through this fn?

this message seems to be coming from what you've removed : 
`        raise errors.EmptyResultException(f'Did not find a workspace with name "{name}".')` 

line 93

Reply:
I think the error is from `CLI version 0.22.0-rc0 is less than master version 0.23.2-dev0. Consider upgrading the CLI.`
I upgraded the CLI, and this is the output:
```
 % det workspace describe testHidden2
Failed to describe workspace: workspace 'testHidden2' not found, please check your permissions.
```
Exactly what we're expecting!